package.json

```diff
@@ -697,9 +732,13 @@
       "@trezor/connect-web>@trezor/connect>@trezor/utxo-lib>tiny-secp256k1": false,
       "@storybook/test-runner>@swc/core": false,
       "@lavamoat/lavadome-react>@lavamoat/preinstall-always-fail": false,
-      "tsx>esbuild": false,
       "@metamask/eth-trezor-keyring>@trezor/connect-web>@trezor/connect>@trezor/protobuf>protobufjs": false,
-      "firebase>@firebase/firestore>@grpc/proto-loader>protobufjs": false
+      "firebase>@firebase/firestore>@grpc/proto-loader>protobufjs": false,
+      "tsx>esbuild": true,
+      "@pmmmwh/react-refresh-webpack-plugin>core-js-pure": true,
+      "@swc/core": true,
+      "@lydell/node-pty": true,
+      "$root$": true
```

Comment:
Turning these on does increase the time it takes `yarn` to complete (note: lavamoat runs these install scripts, not yarn)